### ISSUE TITLE
train label is wrong

### ISSUE THREAD

**Issue by Real-YeJ** [Yuliang-Liu/Curve-Text-Detector#46]
Hello, when I use the ctw-1500 dataset, I found that the the annotation of training set has many mislabeling errors, and the annotation of the test set also has problem. where can I get the correct annotation. Thank you.
